pubspec.yaml

```diff
@@ -9,16 +9,15 @@ environment:
 dependencies:
   args: '>=0.13.7 <3.0.0'
   collection: ^1.14.6
-  http: ^0.13.3
+  http: ">=0.9.0 <0.13.0"
```

Comment:
Do we want to limit this to 0.12 just to save us headaches when we upgrade again soonish?